packages/sketchapp-parser/src/lib/sketch-style-parser.service.ts

```diff
@@ -354,14 +347,14 @@ export class SketchStyleParserService {
   }
 
   transformShapeSolid(node: SketchMSLayer, style: {[key: string]: string}) {
-    const strokeConfig = [];
+    const config = [];
     let offset = 0;
 
     // TODO: Support multiple border
     if (node.style.borders && node.style.borders[0].thickness) {
-      strokeConfig.push(`stroke-width="${node.style.borders[0].thickness / 2}"`);
+      config.push(`stroke-width="${node.style.borders[0].thickness / 2}"`);
```

Comment:
is `node.style.borders[0].thickness` always defined and a number?

Reply:
I'm not actually 100% sure, i will add a guard.